application/plugin_base/terminal.py

```diff
@@ -1,9 +1,11 @@
 import gi
 
-# require specific version of Vte
-gi.require_version('Vte', '2.90')
+from gi.repository import Gtk, Gdk, Vte
 
-from gi.repository import Gtk, Gdk, Vte, GConf
+try:
+	from gi.repository import GConf as gconf
```

Comment:
I pushed a further modification to do this without module renaming.
Christian

On Fri, Apr 29, 2016 at 4:00 PM, Mladen Mijatov <notifications@github.com>
wrote:

> In application/plugin_base/terminal.py
> <https://github.com/MeanEYE/Sunflower/pull/225#discussion_r61591087>:
>
> >
> > -from gi.repository import Gtk, Gdk, Vte, GConf
> > +try:
> > +	from gi.repository import GConf as gconf
>
> I see no reason to import as a different name. In fact, every other import
> has been regular from gi.repository. The fact we are making it a
> conditional import doesn't not seem to warrant making the name lowercase.
>
> As for lowercase, it's a PEP8 recommendation. As comments are not treated
> as sentences, they therefore have no need to start with upper case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MeanEYE/Sunflower/pull/225/files/e577f4d692b5c9fcc6f3bda033cdd85ef27105fc#r61591087>
>